src/main/java/org/example/ReadSaveAndSplitCSV.java

```diff
@@ -0,0 +1,40 @@
+package org.example;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A  class that read and save  the CSV file.;
+ *
+ * @param - file path;
+ * @param - path;
+ */
+public class ReadSaveAndSplitCSV {
+
+    public String readAndSaveCSV() {
+        String filePath = "results.csv";
+        Path path = Paths.get(filePath);
+        byte[] bytes;
+        try {
+            bytes = Files.readAllBytes(path);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
you can stop execution here if you can't read the file. the app is useless without input. don't rethrow this exception